### PR TITLE
Separate SDK Windows try shards into separate builders

### PR DESCRIPTION
**Pull request by keyonghan**
This create separate builder configs for each subshard (Windows try ones).

Related: flutter/flutter#77947

### CODE REVIEW

**Comment by christopherfujino** [comment]
So are these builders redundant duplicates, and once we confirm everything's working you're going to go back and delete the `common.windows_try_builder()` versions?

**Comment by keyonghan** [comment]
> So are these builders redundant duplicates, and once we confirm everything's working you're going to go back and delete the `common.windows_try_builder()` versions?

Yes. The `common.windows_try_builder()` version of shards is equivalent to the newly added separate ones `common.builder_with_subshards`. 
Once populated, I will verify separated builders run successfully before we remove the current consolidated ones.

The workflow will be tracked here: https://github.com/flutter/flutter/issues/77947

File: config/framework_config.star [hunk 1]
```diff
@@ -1213,6 +1258,23 @@ def framework_try_config():
...
+        add_cq = True,
+        list_view_name = list_view_name,
+        properties = {
+            "shard": "tool_integration_tests",
+            "subshards": ["1_3", "2_3", "3_3"],
+            "dependencies": [{"dependency": "android_sdk"}, {"dependency": "chrome_and_driver"}, {"dependency": "open_jdk"}, {"dependency": "goldctl"}],
+        },
+        caches = WIN_DEFAULT_CACHES,
+        os = WINDOWS_OS,
+        # TODO(fujino): https://github.com/flutter/flutter/issues/75003
```
Comment (christopherfujino) [inline config/framework_config.star:1274]: This issue was closed by sharding: https://github.com/flutter/flutter/issues/75003. Can you try removing this TODO and the `execution_timeout = timeout.LONG,`?

**Review by christopherfujino — APPROVED** [review]
1 nit about removing my TODO, otherwise LGTM!

File: config/framework_config.star (hunk 1 above)
Comment (keyonghan) [reply config/framework_config.star]: Removed for both prod and try.

Comment (christopherfujino) [reply config/framework_config.star]: thanks

**Pull request merged**